packages/cli/src/command.ts

```diff
@@ -18,15 +18,47 @@ import { fromString } from 'uint8arrays'
 
 import { config } from './config.js'
 import { createDataModel, loadManagedModel } from './model.js'
+import { SyncOptions } from '@ceramicnetwork/common'
 
 type StringRecord = Record<string, unknown>
 
+// TODO: SYNC_OPTIONS_MAP is also used in js-ceramic/packages/cli. Move this const to '@ceramicnetwork/common'
+export const SYNC_OPTIONS_MAP: { [option: string]: SyncOptions | undefined } = {
+  'prefer-cache': SyncOptions.PREFER_CACHE,
+  'sync-always': SyncOptions.SYNC_ALWAYS,
+  'never-sync': SyncOptions.NEVER_SYNC,
+}
+
 export interface CommandFlags {
   ceramic?: string
   key?: string
   [key: string]: unknown
 }
 
+export const STREAM_ID_ARG = {
+  name: 'streamId',
+  required: true,
+  description: 'ID of the stream',
+}
+
+async function parseSyncOption(input: string): Promise<SyncOptions> {
+  return new Promise((resolve, reject) => {
+    const syncOption = SYNC_OPTIONS_MAP[input]
+    if (syncOption) {
+      resolve(syncOption)
+    } else {
+      reject()
+    }
+  })
+}
+
+export const SYNC_OPTION_FLAG = Flags.integer({
+  char: 's',
+  options: Object.keys(SYNC_OPTIONS_MAP),
+  description: `Controls if the current stream state should be synced over the network or not. 'prefer-cache' will return the state from the node's local cache if present, and will sync from the network if the stream isn't in the cache. 'always-sync' always syncs from the network, even if there is cached state for the stream. 'never-sync' never syncs from the network.`,
+  parse: parseSyncOption
```

Comment:
Could simply be `parse: (input) => SYNC_OPTIONS_MAP[input] ?? SyncOptions.PREFER_CACHE`, no? Any reason the `parseSyncOption` function is async?

Reply:
> Could simply be parse: (input) => SYNC_OPTIONS_MAP[input] ?? SyncOptions.PREFER_CACHE, no?

Yup, updated, thanks :) !

> Any reason the parseSyncOption function is async?

The expected type of `parse` is `(input: number) => Promise<number>`, I used the suggestion that you shared with me offline to just wrap the result in `Promise.resolve()`. Thanks, @PaulLeCam !